content/docs/sidebar.json

```diff
@@ -676,6 +672,21 @@
           }
         ]
       },
+      {
+        "label": "How To",
+        "source": false,
+        "slug": "how-to",
+        "children": [
+          {
+            "slug": "outputs",
+            "label": "Track DVCLive outputs"
+          },
+          {
+            "slug": "checkpoints",
+            "label": "Integrate with DVC checkpoints"
```

Comment:
Maybe expand the how tos to the top-level since there's only 2. Don't even need to call them "How to". Idea:
**Tracking DVCLive Outputs**
**Using DVC Checkpoints**